pkg/job/update_user_roles.go

```diff
@@ -112,15 +124,15 @@ func (c *updateUserRoles) updateLevelRoles(guildID string) error {
 		"rolesToRemove": rolesToRemove,
 	}).Info("entity.RemoveGuildMemberRoles executed successfully")
 
-	if err := c.entity.AddGuildMemberRoles(guildID, rolesToAdd); err != nil {
+	if err := c.entity.AddGuildMemberRoles(guildID, guild.LogChannelID, rolesToAdd, rolesToRemove); err != nil {
```

Comment:
why need pass `rolesToRemove` to `AddGuildMemberRoles()`?

Reply:
Because update func used map of roles base on userID to update role, so it's hard to track state update role for each user to create msg to log channel, therefore may need to pass `rolesToRemove` for create msg to log channel more conveniently